ui/src/components/erc/Metadata/Metadata.js

```diff
@@ -33,29 +43,36 @@ class Metadata extends React.Component {
     render() {
         const positions = [this.metadata.spatial.union.bbox[0], this.metadata.spatial.union.bbox[1], this.metadata.spatial.union.bbox[2], this.metadata.spatial.union.bbox[3]]
         const languages = this.metadata.languages ? languageCodes.filter(language => this.metadata.languages.includes(language.code)).map(language => language.full).toString().replace(/,/g, ", ") : null;
-        const keywords  = this.metadata.keywords ? this.metadata.keywords.toString().replace(/,/g, ", ") : null;  
+        const keywords  = this.metadata.keywords ? this.metadata.keywords.toString().replace(/,/g, ", ") : null; 
+
         return (
             <>
                 <div style={{ width: "90%", marginLeft: "5%", textAlign: "justify" }}>
                     <h2>Metadata of compendium:</h2>
                     <span id="title"><b>Title: </b>{this.metadata.title}</span> <br />
                     <p> <span style={{ "font-weight": "bold" }}> Created on: </span> {this.props.erc.created.substr(0, 10)} {this.props.erc.created.substr(11, 5)}  <br />
-                        <span style={{ "font-weight": "bold" }}> by: </span> {this.props.erc.user} </p>
+                        <span style={{ "font-weight": "bold" }}> by: </span>
+                        <a href ={"https://orcid.org/"+this.props.erc.user} target="_blank" >
+<img alt="ORCID logo" src="https://info.orcid.org/wp-content/uploads/2019/11/orcid_16x16.png" width="16" height="16" />
```

Comment:
Do we often use images form other domains? I suggest to add this very small image to our own resources and load them from the local host.